src/ucs/sys/topo/base/topo.c

```diff
@@ -345,30 +345,37 @@ ucs_topo_get_distance_sysfs(ucs_sys_device_t device1,
     /* If one of the devices is unknown, we assume near topology */
     if ((device1 == UCS_SYS_DEVICE_ID_UNKNOWN) ||
         (device2 == UCS_SYS_DEVICE_ID_UNKNOWN) || (device1 == device2)) {
-        *distance = ucs_topo_default_distance;
-        return UCS_OK;
+        goto default_distance;
     }
 
     status = ucs_topo_sys_dev_to_sysfs_path(device1, path1, sizeof(path1));
     if (status != UCS_OK) {
-        return status;
+        ucs_trace("failed to get sysfs path for %s",
```

Comment:
maybe better debug, as it is not supposed to be a frequent message?

Reply:
`ucs_topo_get_distance` is called from `ucp_proto_common_get_lane_distance`, which is called from other methods: ucp_proto_rndv_ctrl_select_remote_proto, ucp_proto_common_get_lane_perf...
Is it still worth to make the level of verbosity debug?

Reply:
Changed to debug.